monitor.py

```diff
@@ -1203,26 +1195,26 @@ def get_detailed_job_info(prow_ci_name,prow_ci_link,start_date=None,end_date=Non
 
         print("Nightly info-", nightly)
         
-        if job_status == 'SUCCESS':
+        if build_status == 'SUCCESS':
             deploy_count += 1
             e2e_count=e2e_count+1
-            if "sno" not in job:
+            if "sno" not in build:
                 print("Lease Quota-", lease)
-            check_node_crash(job)
+            check_node_crash(build)
             print("This is a Green build")
```

Comment:
```suggestion
            print("Job passed")
```

Reply:
@Shilpa-Gokul I think **"Build Passed"** message will be more appropriate, because according to our CoreOS CI a "4.12-periodic" is a job name and that job will have multiple builds with different build ids.